screen/index.html

```diff
@@ -78,5 +78,74 @@ <h1 class="primary__title">Get More Done with whitepace</h1>
         </div>
       </section>
     </header>
+     <main>
+      <section class="feature">
+        <div class="container">
+          <div class="flex-grid-item">
+            <div class="feature__content layout">
+              <h2 class="primary__title">
+                Project <br />
+                <span class="feature__title-highlight">Management</span>
+              </h2>
```

Comment:
Management là nội dung heading. Không đặt trong span nhé em. 
- Sử dụng class vào thẻ h2 rồi style. Xử lý tương tự với các title sau cần dùng hiệu ứng này.
```suggestion
               <h2 class="primary__title title-highlight">
                Project <br />
                Management
              </h2>
```